client/src/components/header/NotificationItem.vue

```diff
@@ -0,0 +1,140 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div
+    class="element-container"
+    :class="{
+      unread: notification.read === false
+    }"
+    @mouseover="$emit('mouseOver', notification)"
+  >
+    <ion-item
+      class="element ion-no-padding"
+      @click="onClick"
+    >
+      <div class="element-type">
+        <div class="element-type__icon">
+          <ms-image
+            :image="getFileIcon('test.doc')"
```

Comment:
Where does this icon come from? What lets you chose what to display?

Reply:
it's because it's only an example of notification atm, the type management is missing